migrations/migration-9/index.js

```diff
@@ -40,7 +40,7 @@ async function pinsToDatastore (blockstore, datastore, pinstore, onProgress) {
   for await (const cid of pinset.loadSet(blockstore, pinRoot, PinTypes.recursive)) {
     counter++
 
-    /** @type {{ depth: number, version?: CID.CIDVersion, codec?: CodecCode }} */
+    /** @type {{ depth: number, version?: 0 | 1, codec?: number }} */
```

Comment:
nit: perhaps worth adding a type for cidVersion? This happens in multiple places here and will probably happen the same in other repos

Reply:
@Gozala is it worth adding this to multiformats? will probably need an interfaces.ts in src (we don't have one yet). We have `Code extends number` in https://github.com/multiformats/js-multiformats/blob/master/src/codecs/interface.ts and we could also pull that up too?

Reply:
The older `cids` module exported the `CIDVersion` type which was useful but it doesn't appear to have an equivalent in the new `multiformats` module.

I've opened https://github.com/multiformats/js-multiformats/pull/87 to add it.

Reply:
This uses the exported `CIDVersion` from `multiformats@9.x.x` now.